nav/load_waypoints/scripts/navigate_waypoints.py

```diff
@@ -100,7 +106,7 @@ def wait_for_utm_transform(self):
     
     def get_next_waypoint(self):
         waypoint = self.waypoints[self.curr_waypoint_idx]
-        print self.curr_waypoint_idx
+        print(self.curr_waypoint_idx)
         self.curr_waypoint_idx += 1
```

Comment:
Could you make the waypoints wrap around? We might need this for the second lap

Reply:
Essentially no change other than style now

Reply:
Wrap around to prevent crashing. On the final index, we'll get index out of bounds

Also, is the print change correct? Python2 expects the old change

Reply:
I think I did it cause I had backend python3 default for some time. I don't think ( ) will make a diff even in python2